tests/test_mongodb_id_pk_variations.py

```diff
@@ -108,7 +113,8 @@ def setUp(self):
             client["simple_db"]["coll_with_date_id"].insert_many(generate_docs_date_id())
             client["simple_db"]["coll_with_32_bit_int_id"].insert_many(generate_docs_32_bit_int_id())
             client["simple_db"]["coll_with_64_bit_int_id"].insert_many(generate_docs_64_bit_int_id())
-            client["simple_db"]["coll_with_128_decimal_id"].insert_many(generate_docs_128_decimal_id())
+            # cannot have decimal128 is not a supported replication key type
+            #client["simple_db"]["coll_with_128_decimal_id"].insert_many(generate_docs_128_decimal_id())
```

Comment:
Not clear on why this is removed. 
1. Does the primary key need to also be the replication key for this test, shouldn't the focus be on the primary key?
2. Where does the 'not supported' limitation come from? Is it the tap, the source db, the client used in the test?
3. If the datatype does not apply to this scenario, can it just be removed from the test entirely?

Reply:
The limitation is from the tap, when we select _id to be decimal128 the tap throws an error that it is not a supported replication key type. So this datatype does not apply to this scenario.

Reply:
yeah so i am removing it from the test